WORKSPACE

```diff
@@ -32,8 +32,8 @@ bazel_skylib_workspace()
 
 git_repository(
     name = "com_google_protobuf",
-    remote = "https://github.com/protocolbuffers/protobuf",
     commit = "655310ca192a6e3a050e0ca0b7084a2968072260",
+    remote = "https://github.com/protocolbuffers/protobuf",
```

Comment:
Nit: This should use `http_archive` and not `git_repository`.

Reply:
I am dubious of the value of http_repository.  If github is compromised we are already in big trouble.